internal/api/provider/azure.go

```diff
@@ -2,30 +2,34 @@ package provider
 
 import (
 	"context"
-	"errors"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"regexp"
 	"strings"
 
+	"github.com/coreos/go-oidc/v3/oidc"
 	"github.com/supabase/gotrue/internal/conf"
 	"golang.org/x/oauth2"
 )
 
 const IssuerAzure = "https://login.microsoftonline.com/common/v2.0"
+const IssuerAzureMicrosoft = "https://login.microsoftonline.com/9188040d-6c67-4c5b-b112-36a304b66dad/v2.0"
```

Comment:
Might have missed it in the docs but what's the significance of `9188040d-6c67-4c5b-b112-36a304b66dad`? Is it an arbitrary endpoint?

Reply:
Looks like it's from here-  https://learn.microsoft.com/en-us/azure/active-directory/develop/id-token-claims-reference#payload-claims

Can consider adding a comment if there isn't already one elsewhere